lib/matplotlib/backends/backend_qt.py

```diff
@@ -517,7 +517,12 @@ def _draw_idle(self):
             if not self._draw_pending:
                 return
             self._draw_pending = False
-            if self.height() <= 0 or self.width() <= 0:
+            try:
+                if self.height() <= 0 or self.width() <= 0:
+                    return
+            except RuntimeError:
+                # This can happen if the c++ object is already cleaned up
```

Comment:
Can we check the error message to ensure we’re not swallowing other potential RuntimeErrors?

Also, are we sure this is the only place that may access the C++ object?